slack/slack/lucky-draw/rust/src/lucky-draw.rs

```diff
@@ -0,0 +1,27 @@
+#[allow(unused_imports)]
+use wasmedge_bindgen::*;
+use wasmedge_bindgen_macro::*;
+use rand::{thread_rng, Rng};
+
+#[wasmedge_bindgen]
+pub fn run(s: String) -> String {
+    let mut rng = thread_rng();
+
+    match s.as_str() {
```

Comment:
You can use `to_lowercase()` instead of enumerate the case.

Reply:
sure, thanks

Reply:
No changes here yet